### DISCUSSION TITLE
Link to the `Iterator` broken in the Readme

### DISCUSSION THREAD

**Discussion by celestix** [abseil/abseil-cpp#1414] [General]
Readme documentations mention `Iterator` in the library component and with an embedded hyperlink, but when you try to open it, it says not found as there is no `iterator` directory in the source code.

**Comment by derekmauro** [comment]
This is a library we never released and don't plan to release, but somehow it ended up in the documentation. I will remove it.

**Reply by celestix** [reply]
> This is a library we never released and don't plan to release, but somehow it ended up in the documentation. I will remove it.

Oh, okay. Thanks for the reply!
